zebra-state/src/service/finalized_state.rs

```diff
@@ -428,7 +428,7 @@ impl FinalizedState {
 
             let mut blocks_size_to_dump = match self.network {
                 Network::Mainnet => MAINNET_AWAY_FROM_TIP_BULK_SIZE,
-                Network::Testnet => TESTNET_AWAY_FROM_TIP_BULK_SIZE,
+                Network::Testnet(_) => TESTNET_AWAY_FROM_TIP_BULK_SIZE,
             };
```

Comment:
This code is incorrect, and will fail if the public or custom testnet has a significant number of 2MB blocks. Instead, the value of `MAINNET_AWAY_FROM_TIP_BULK_SIZE` should be used for all networks.